.gitignore

```diff
@@ -1,8 +1,9 @@
 .idea
 scripts/bin
-tests/src/**/*.js
+**/*.js
```

Comment:
this is risky.
we might need to use plain javascript files someday

Reply:
ok. I don't remember why i did this